app/helpers/maintenance_tasks/tasks_helper.rb

```diff
@@ -44,12 +44,13 @@ def progress(run)
 
       progress = Progress.new(run)
 
-      tag.progress(
+      progress_bar = tag.progress(
         value: progress.value,
         max: progress.max,
-        title: progress.title,
```

Comment:
Is there accessibility value in continuing to have a title of some sort on the progress bar?

Reply:
Moving the text into a `<span>` and out of the title is probably a _plus_ on the accessibility side of things - from https://accessible-app.com/pattern/vue/progress:

> Using progress element alone is not the worst thing you could do, but there are always way to improve. If you think, for example, that the presence of a <progress> bar alone leads to automatic screen reader output, when it updates – nope, that is not the case. A screen reader's virtual cursor has to be placed on the progressbar in order to hear its information. 

Whereas I believe pulling the text out into a span makes it a readable element. 

We could keep the title with the same value, but I'm not sure it's worth having that information duplicated 🤷‍♀️